packages/db/src/query/live/collection-subscriber.ts

```diff
@@ -11,6 +11,8 @@ import type { OrderByOptimizationInfo } from "../compiler/order-by.js"
 import type { CollectionConfigBuilder } from "./collection-config-builder.js"
 import type { CollectionSubscription } from "../../collection/subscription.js"
 
+const loadMoreCallbackSymbol = Symbol(`tanstack.db.loadMore`)
```

Comment:
nit: I propose we change this one to `@tanstack/db.loadMore` to be consistent with the naming we used in `collection-registry.ts`:
```ts
const BUILDER_SYMBOL = Symbol.for(`@tanstack/db.collection-config-builder`)
```